REAMDE

```diff
@@ -0,0 +1 @@
+README.md
```

Comment:
Ah, that's a symlink. I've never seen one in GitHub diff; that's new for me.

Honestly I'd prefer to not bother the project with symlinks. You know, we still have a plenty of troubles with them on Windows even in 2017 😞 (and Windows is still a major development and environmental platform for WPF-Math, even if we're as willing to go cross-platform as we are).

I think I'll delete these symlinks if that's okay for you, and fix the testing/publishing scripts.